.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+    "trailingComma": "none",
+    "tabWidth": 4,
+    "semi": true,
+    "singleQuote": true,
```

Comment:
SingleQuote is set to true here, but it looks like all the quotes throughout the app are double quotes. Run prettier and switch to single quotes.